src/redux/configureStore.js

```diff
@@ -0,0 +1,17 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import regions from './regions/regions';
+// import missionsReducer from './missions/missions';
+
+const reducer = combineReducers({
+  regions,
+  // missionsReducer,
```

Comment:
Let's remove commented codes, Kindly check for other occurrences 🙂

Reply:
Just deleted all the comments, thanks for the review!